src/test/java/math/Vector3fTest.java

```diff
@@ -0,0 +1,27 @@
+package math;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class Vector3fTest {
```

Comment:
What about other methods? Add tests for more of them.

Test method names should include 'test' in them.